### PR TITLE
change model output parameter to last_hidden_states

### PR DESCRIPTION
**Pull request by jiqing-feng**
`AutoModel` without any `Head` will output `hidden_state` instead of `logits`。

### CODE REVIEW

File: optimum/intel/ipex/modeling_base.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def forward(
             inputs["token_type_ids"] = token_type_ids
 
         outputs = self._call_model(**inputs)
-        return ModelOutput(**outputs) if isinstance(outputs, dict) else ModelOutput(logits=outputs[0])
+        return ModelOutput(**outputs) if isinstance(outputs, dict) else ModelOutput(last_hidden_state=outputs[0])
```
Comment (echarlaix) [inline optimum/intel/ipex/modeling_base.py:196]: it should be `logits` for all other classes (like `IPEXModelForSequenceClassification`, `IPEXModelForTokenClassification`, ...) can you update it accordingly ?

Comment (jiqing-feng) [reply optimum/intel/ipex/modeling_base.py]: Add a class parameter `output_name` would be more convenient and clean. Please take a review and let me know your opinion:)

**Review by echarlaix — APPROVED** [review]
Thanks for the fix @jiqing-feng !
